CMakeLists.txt

```diff
@@ -286,13 +287,26 @@ IF (BALL_HAS_VIEW)
 
 	ADD_DEFINITIONS(${Qt5Widgets_DEFINITIONS})
 
-        LIST(APPEND VIEW_DEP_LIBRARIES Qt5::Widgets
-                                       Qt5::OpenGL
-                                       Qt5::PrintSupport
-                                       Qt5::Network
-                                       Qt5::Test
-                                       Qt5::Xml)
+	LIST(APPEND VIEW_DEP_LIBRARIES Qt5::Widgets
+	                               Qt5::OpenGL
+	                               Qt5::PrintSupport
+	                               Qt5::Network
+	                               Qt5::Test
+	                               Qt5::Xml)
+
+	IF(USE_QTWEBENGINE)
+		FIND_PACKAGE(Qt5WebEngineWidgets REQUIRED)
```

Comment:
I also had to add a `FIND_PACKAGE(Qt5WebEngine)` here. Also I think that REQUIRED should be removed, as we can build BALL without the WebEngine.